src/Fabulous.AST.Tests/OpenDirectives/OpenTests.fs

```diff
@@ -0,0 +1,41 @@
+namespace Fabulous.AST.Tests
+
+open FSharp.Compiler.Text
+open Fantomas.Core
+open Fantomas.Core.SyntaxOak
+open NUnit.Framework
+
+open Fabulous.AST
+
+open type Ast
+
+module OpenTests =
+
+    [<Test>]
+    let ``Produces a simple open directive from a string`` () =
+        AnonymousModule() { Open("ABC") }
+        |> produces
+            """
+
+open ABC
+
+"""
+
+    [<Test>]
+    let ``Produces an open directive from a node`` () =
+        AnonymousModule() {
+            Open(
+                IdentListNode(
```

Comment:
This is both beautiful and a missing opportunity.
I think having some sort of helper that takes `"ABC.DEF"` as input and converts it to an `IdentListNode` would also be very useful.

Any thoughts on auxiliary functions? Perhaps operators although those have a downside as well.

Reply:
@nojaf . We could add another widget constructor, `Open(["ABC"; "DEF"])` and we do the conversion

Reply:
That's a good suggestion. We can definitely have a small parser that could convert a string into a IdentListNode (why not even use Fantomas on the fly to get us the AST of that specific string if we give it the context somehow)